app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
```

Comment:
```suggestion
    @user = User.new user_params

    if @user.save
```
chỗ nào bỏ dc ngoặc thì bỏ ngoặc e nhé, k khuyến khích dùng ngoặc